test/convergence/test_mini_models.py

```diff
@@ -363,6 +363,10 @@ def run_mini_model(
         ("mini_phi3", 32, 1e-4, torch.bfloat16, 1e-8, 1e-5, 1e-2, 1e-5, 1e-2, 1e-5),
     ],
 )
+@pytest.mark.skipif(
+    torch.cuda.get_device_capability()[0] < 8,
```

Comment:
For maintainability, should we wrap this and following conditions in a function within `test.utils`?